source/Pages/compute_on_spider.rst

```diff
@@ -221,17 +225,26 @@ Job types
   * For example, by specifying ``-c 2`` you request 2 cores and 160 GB scratch disk
   * When you target specifically our fat nodes with 12TB available scratch, the provided scratch disk per requested core is 200 GB
 
+GPU jobs
+========
+* For jobs that require GPU resources a specific partition is available (see :ref:`partitions <partitions>` for all the different partitions).
+
+
+.. _partitions:
+
 ================
 Slurm partitions
 ================
 
-We have configured four partitions on :abbr:`Spider (Symbiotic Platform(s) for Interoperable Data
+We have configured four CPU and two GPU partitions on :abbr:`Spider (Symbiotic Platform(s) for Interoperable Data
```

Comment:
perhaps also provide a link to refer back to the description of these partitions (above)